containenv/containenv/cli/commands/init.py

```diff
@@ -21,9 +21,25 @@
 Options:
   -h, --help        Display this help message and exit.
 '''
+import collections
+import docker
 import os
+import logging
+from pkg_resources import resource_filename
```

Comment:
I don't think it's standard in Python 3.x either, although I'm not entirely sure. It's part of `setuptools` which just seems ubiquitous.

`distutils` is part of the standard library, but `setuptools` is a separate library.